packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/lib/graphql-client-factory/create.ts

```diff
@@ -16,11 +16,13 @@ export const createGraphQLClientFactory = (config: JssConfig) => {
   if (config.sitecoreEdgeContextId) {
     clientConfig = {
       endpoint: getEdgeProxyContentUrl(config.sitecoreEdgeContextId, config.sitecoreEdgeUrl),
+      xmCloud: true,
     };
   } else if (config.graphQLEndpoint && config.sitecoreApiKey) {
     clientConfig = {
       endpoint: config.graphQLEndpoint,
       apiKey: config.sitecoreApiKey,
+      xmCloud: true,
```

Comment:
I think that graphql-request-client doesn't really depend on the SXP vs XMCloud environment, it's an abstract request functionality. So we shouldn't define this option, there are no specific use cases

Reply:
graphql-request-client does not depend on it - but this was my attempt to enabled xmCloud-only site query for xmcloud addon. 
 I'll try the ejs approach instead